test/nerves_hub_link/update_manager_test.exs

```diff
@@ -1,12 +1,16 @@
 defmodule NervesHubLink.UpdateManagerTest do
-  use ExUnit.Case
+  use ExUnit.Case, async: false
+
   alias NervesHubLink.{FwupConfig, UpdateManager}
   alias NervesHubLink.Message.{FirmwareMetadata, UpdateInfo}
   alias NervesHubLink.Support.FWUPStreamPlug
 
+  import Mock
+
   describe "fwup stream" do
     setup do
-      port = 6000
+      port = Enum.random(6000..6999)
```

Comment:
Random here but specific in another test?

Reply:
There was an error in a Circle run where a test failed due to the port being in use, which is odd as the test was not async, but I added this as a way to make sure it didn't happen again.